FamilyNet/FamilyNet/Controllers/OrphanagesController.cs

```
@@ -245,5 +249,34 @@ public IActionResult SearchOrphanageOnMap()
 
             return View(orphanages);
         }
+
+        private Tuple<float?, float?> GetCoordProp(Address address)
+        {
+            float? X = null;
+            float? Y = null;
+
+            try
+            {
+                var nominatim = new Nominatim.API.Geocoders.ForwardGeocoder();
+                var d = nominatim.Geocode(new Nominatim.API.Models.ForwardGeocodeRequest()
+                {
+                    Country = address.Country,
+                    State = address.Region,
+                    City = address.City,
+                    StreetAddress = String.Concat(address.Street, " ", address.House)
+                });
+
+                //TODO:some validation for search
+
+                X = (float)d.Result[0].Latitude;
+                Y = (float)d.Result[0].Longitude;
+            }
+            catch (Exception ex)
+            {
+
```

add something, empty catch != good